### ISSUE TITLE
rvd log is created without any permissions set

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#70]
```sh
$ grep -n log ../etc/rvd.json 
8:  "log": "/var/log/rvd.log",
$ ls -la "/var/log/rvd.log"
----------  1 root  wheel  1999 Sep  1 15:26 /var/log/rvd.log
```
Can you please ensure that the log is readable for the owner? Thanks

**Comment by erikbor** [comment]
something like:
```sh
-rw-------  1 root  wheel  1999 Sep  1 15:26 /var/log/rvd.log
```

**Comment by jjr840430** [comment]
fixed on #74
